lib/galaxy/dependencies/pipfiles/default/pinned-dev-requirements.txt

```diff
@@ -2,16 +2,16 @@
 --extra-index-url https://pypi.python.org/simple
 alabaster==0.7.11
 argh==0.26.2
-atomicwrites==1.1.5
-attrs==18.1.0
+atomicwrites==1.2.1; python_version != '3.3.*'
+attrs==18.2.0
 babel==2.6.0
-certifi==2018.8.13
+certifi==2018.8.24
 chardet==3.0.4
 commonmark==0.5.4
 docutils==0.14
-funcsigs==1.0.2; python_version < '3.3'
+funcsigs==1.0.2; python_version == '2.6'
```

Comment:
This (and others below) looks like a mistake from pipenv which should be automatically fixed by `lib/galaxy/dependencies/pipfiles/update.sh` (which is what `make update-dependencies` run).

Reply:
You may also noticed that running `make update-dependencies` has down-graded both cloudbridge and genomespace to their previous versions (those already on dev). Any thoughts?

Reply:
That's because `python-genomespaceclient` is pinned to `<1.0` in lib/galaxy/dependencies/pipfiles/default/Pipfile ,see https://github.com/galaxyproject/galaxy/pull/6668/files#diff-3c6aba54d3aab2581826ac1eef590665L72

Reply:
Thanks! then I think it might be better hold this PR till #6668 is updated and merged.